content/blog/2023-03-22-etc-and-bitcoin-have-the-fifthening-and-the-halving-in-2024/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "ETC and Bitcoin Have the Fifthening and the Halving in 2024"
+date: 2023-03-22
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["education", "announcement"]
+linkImage: ./etc-fifthening-banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JaNkywXLg2M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+## What is the Fifthening in ETC?
+
+![The ETC fifthening.](./1.png)
+
+Ethereum Classic’s monetary policy has a rule that every 5,000,000 blocks it will reduce the mining reward by 20%. This is called the fifthening because it is a 1/5 cut in the issuance of the coin.
+
+The mining reward is the amount of ETC that is issued to pay for miners to produce blocks for the network.
+
+In ETC, a block is currently being produced every 13.1525 seconds on average.
+
+The historial schedule of payments per block to miners by the Ethereum Classic network has been the following:
+
+**Era 1:** 5.00 ETC, from block 0, which happened on June 30, 2015
+
+**Era 2:** 4.00 ETC, from block 5,000,000, which happened on December 11, 2017
+
+**Era 3:** 3.20 ETC, from block 10,000,000 which happened on March 17, 2020
+
+**Era 4:** 2.56 ETC, from block 15,000,000 which happened on April 25, 2022
+
+Because ETC is now on block 17,088,524 as of this writing, it is paying 2.56 ETC per block as we are in era 4.
+
+In the next era, which will be era 5, the block reward will be reduced to 2.048 ETC.
+
+## When Will ETC’s Next Fifthening Happen?
+
+To estimate when the next ETC fifthening will happen we need to know how many blocks are left to the next fifthening. Then, we need to multiply that number of blocks by the average block time of the network in seconds. This will give us the number of seconds left to the next fifthening. If we transform that amount into days, then we will be able to add the result to the current date and determine when the next fifthening will happen.
```

Comment:
Line 43: Correct preposition usage
```suggestion
To estimate when the next ETC fifthening will happen we need to know how many blocks are left until the next fifthening. Then, we need to multiply that number of blocks by the average block time of the network in seconds. This will give us the number of seconds left until the next fifthening. If we transform that amount into days, then we will be able to add the result to the current date and determine when the next fifthening will happen.
```